build_tools/circle/build_test_pypy.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+apt-get -yq update
+apt-get -yq install libatlas-dev libatlas-base-dev liblapack-dev gfortran
+
+pip install virtualenv
+
+if command -v pypy3; then
+    virtualenv -p $(command -v pypy3) pypy-env
+elif command -v pypy; then
+    virtualenv -p $(command -v pypy) pypy-env
+fi
+
+source pypy-env/bin/activate
+
+python --version
+which python
+
+pip install --extra-index https://antocuni.github.io/pypy-wheels/ubuntu numpy==1.14.4 Cython Tempita pytest
```

Comment:
we need tempita?

Reply:
You are right, this appears to work without tempita. Removed it.

(Maybe because we are using binary wheels. I think I added it initially as I was getting tempita import error without it, but can't remember where anymore.)